ts/features/bonus/cgn/screens/merchants/CgnMerchantsCategoriesSelectionScreen.tsx

```diff
@@ -62,7 +75,7 @@ const CgnMerchantsCategoriesSelectionScreen = () => {
 
   const categoriesToArray: ReadonlyArray<ProductCategoryEnum | "All"> = [
     "All",
-    ...Object.entries(ProductCategoryEnum).map(value => value[1])
+    ...(isReady(remoteCategories) ? remoteCategories.value : [])
```

Comment:
If the `cgnCategories.request` fails (`cgnCategories.failure`) no feedback is provided to the user. Is this intentional?

Reply:
Actually didn't thought about a feedback since this isn't actually blocking the user from listing the merchants clicking "All the categories"